sandbox-search.js

```diff
@@ -11,13 +11,28 @@ export default async function () {
   const text = await readFile('sandbox-seed.json', { encoding: 'utf-8' })
   const { circulars, synonyms } = JSON.parse(text)
 
+  const groupedSynonyms = {}
+  synonyms.forEach((synonym) => {
+    if (groupedSynonyms[synonym.synonymId]) {
+      groupedSynonyms[synonym.synonymId].push(synonym.eventId)
+    } else {
+      groupedSynonyms[synonym.synonymId] = [synonym.eventId]
+    }
+  })
```

Comment:
Is there a builtin JavaScript method or a Lodash helper to group an iterable by the value of a key function?